package.json

```diff
@@ -1,10 +1,20 @@
 {
+  "scripts": {
+    "test-accessibility-all-gui": "npx cypress open",
```

Comment:
If you want to define these kinds of shortcuts (personally I'm not convinced they're all necessary), the could the order of the command parts be changed slightly? I would prefer e.g. `test-accessibility-gui-all` instead of the above. This way it's easy to delete `-all` from the end and change to e.g. `-about`. It takes more work (more keystrokes) if you have to edit the middle part of the command.